subsys/testsuite/ztest/src/ztest.c

```diff
@@ -33,6 +33,11 @@ static bool failed_expectation;
 
 #define NUM_ITER_PER_SUITE CONFIG_ZTEST_SHUFFLE_SUITE_REPEAT_COUNT
 #define NUM_ITER_PER_TEST  CONFIG_ZTEST_SHUFFLE_TEST_REPEAT_COUNT
```

Comment:
You may want to guard these with an `#ifndef CONFIG_ZTEST_REPEAT` to prevent the macro re-definition warning.

Reply:
That's right, I have only been following @golowanow 's suggestions.